src/Client/ClientInterfaceRule.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * MIT License
+ * This file is part of the Spryker Suite.
```

Comment:
wrong content! I guess we need to use keep the core one then to have https://github.com/spryker/spryker/blob/master/Bundles/ProductBundle/src/Spryker/Client/ProductBundle/Grouper/ProductBundleGrouperInterface.php#L3-L6 output

Please then use this content inside a .license file.